src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -2762,14 +2762,14 @@ try {
     foreach ($requests as $request) {
         $stream->write($request);
         // if required, read a single response from the stream
-        $element = $stream->read();
-        // doSomethingWith($element)
+        $responseItem = $stream->read();
+        print($responseItem);
     }
     $stream->closeWrite();
-    $element = $stream->read();
-    while (!is_null($element)) {
-        // doSomethingWith($element)
-        $element = $stream->read();
+    $responseItem = $stream->read();
+    while (!is_null($responseItem)) {
+        print($responseItem);
```

Comment:
`print($x)` works for primitive types, but produces an error when called on arrays or protobuf objects.

Reply:
Thanks for the quick response! I'll have a PR to move code from sample_common to standalone_sample first to not affect the in method samples. 

For the `print($x)` part, would changing all of them to `var_dump($x)` make sense here?

Reply:
Yes. This is a feature requested by DPEs. The idea is here to allow post-processing the response from an API call (printing, defining the return value to a local variable or adding extra comments, etc.) by adding configuration in gapic.yaml. In a normal case, `$x` would never be an array (arrays are handled separately using loops(https://github.com/googleapis/gapic-generator/pull/2312/files/08083d209fdca8da222ef61fd87881e92bcd6da4#diff-36f32ff1b2217a381ca1eaeeea45dfe6R2865)), but it's possible `$x` is a protobuf object.

Reply:
OK cool. I think `print_r` is preferable to `var_dump`, although I don't have a strong opinion, and the real solution is to make the protobuf objects behave correctly when passed to `print`. For now I would use `print_r`.

Reply:
(If you need to convert a protobuf object to a string when working with printf statements, then you could use print_r($x, true), which will return a string instead of printing.)